upup/pkg/fi/cloudup/awsup/aws_cloud.go

```diff
@@ -618,10 +617,13 @@ func awsBuildCloudInstanceGroup(c AWSCloud, ig *kops.InstanceGroup, g *autoscali
 			glog.Warningf("ignoring instance with no instance id: %s in autoscaling group: %s", id, cg.HumanName)
 			continue
 		}
-		currentConfigName, err := findInstanceLaunchConfiguration(i)
-		if err != nil {
-			return nil, err
+		// @step: check if the instance is terminating
+		if aws.StringValue(i.LifecycleState) == autoscaling.LifecycleStateTerminating {
+			glog.Warningf("ignoring instance  as it is terminating: %s in autoscaling group: %s", id, cg.HumanName)
+			continue
 		}
+		currentConfigName := findInstanceLaunchConfiguration(i)
```

Comment:
We now take the configuration name regardless, if empty, we just continue and mark as NeedUpdate ... in effect what would happen [here](https://github.com/kubernetes/kops/blob/1.11.1/upup/pkg/fi/cloudup/awsup/aws_cloud.go#L551-L575)